packages/jmix-front-generator/src/building-blocks/stages/writing/pieces/i18n.ts

```diff
@@ -78,11 +80,18 @@ function mergeI18nMessages(
   }
 
   return hasNewEntries(componentMessages, existingMessages)
-    ? {...componentMessages, ...existingMessages}
-    : null;
+    ? {
+      ...componentMessages,
+      ...(
+        typeof existingMessages === 'object'
+          ? existingMessages
+          : {}
+      )
```

Comment:
What is the point of that? 

This will have the same effect when `existingMessages` is `null` or `undefined`

```
{
  ...componentMessages,
  ...existingMessages
}
```

Reply:
This is needed for ts compiler. Behavior is the same.